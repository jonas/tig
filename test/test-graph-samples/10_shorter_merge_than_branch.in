commit A H B C D J E
    Commit A - merge H, B, C, D, and E
commit B K
    Commit B - after K
commit C H
    Commit C - after H
commit J I
    Commit J - after I
commit D K
    Commit D - after K
commit E F
    Commit E - after F
commit K F
    Commit K - after F
commit F G H
    Commit F - merge G and H
commit G I
    Commit G - after I
commit H I
    Commit H - after I
commit I
    Commit I
