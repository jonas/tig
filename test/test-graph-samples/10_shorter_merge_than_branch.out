●─┬─┬─┬─┬─╮ Commit A - merge H, B, C, D, and E
│ ● │ │ │ │ Commit B - after K
│ │ ● │ │ │ Commit C - after H
│ │ │ │ ● │ Commit J - after I
│ │ │ ● │ │ Commit D - after K
│ │ │ │ │ ● Commit E - after F
│ ●─│─╯ │ │ Commit K - after F
│ ●─│─╭─╯─╯ Commit F - merge G and H
│ ● │ │ ╭─╯ Commit G - after I
●─│─┴─│─╯ Commit H - after I
◎─┴───╯ Commit I
